frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.test.tsx

```diff
@@ -0,0 +1,140 @@
+import React from 'react';
+import { render, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
```

Comment:
I'm sorry that I did not catch this on my first review, but we should setup userEvent a little differently now. The lastest version of user-event is backwards compatible, but in future versions, this approach will break.

We need to use `userEvent.setup()` within each tests instead of using the `userEvent` directly form the import. 

Example:
```
it('should be written like following:', () => {
   const user = userEvent.setup(); // <-- This line is what Im talking about.
   await user.click('some-button');
});
```

Please read more at the docs here: 
https://testing-library.com/docs/user-event/intro/#writing-tests-with-userevent

Reply:
No worries! 😄